HibernateOrmRestDataPanacheTest is generating a false negative in a non english locale

## Describe the bug
I'm using macOS with a polish language/locale set - when I tried to build Quarkus (including tests), it failed at `HibernateOrmRestDataPanacheTest`, with the following output:

```
[ERROR] Failures:
[ERROR]   HibernateOrmRestDataPanacheTest.shouldNotCreateBookWithBlankTitle:183 1 expectation failed.
JSON path parameterViolations[0].message doesn't match.
Expected: must not be blank
  Actual: nie może być odstępem
```

FYI: the "nie może być odstępem" means exactly "must not be blank".

### Expected behavior
Tests are passing regardless of which locale is set.

### Actual behavior
Tests are passing in the case when the English locale is set.


## To Reproduce

Steps to reproduce the behavior:
1. Switch your OS language/locale to a non-EN one (might require a restart)
2. Run `HibernateOrmRestDataPanacheTest` (it should fail)

## Environment (please complete the following information):
LANG=pl_PL.UTF-8

### Output of `java -version`
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment (build 13.0.2+8)
OpenJDK 64-Bit Server VM (build 13.0.2+8, mixed mode, sharing)

### GraalVM version (if different from Java)

### Quarkus version or git rev
787bc2c618028e3812897d25b95558c5526245fe

I will be happy to take care of this issue, as i.e. it could be fixed by either:

1.) Setting a default locale in [`application.properties`](https://github.com/quarkusio/quarkus/blob/main/integration-tests/hibernate-orm-rest-data-panache/src/main/resources/application.properties), as described here: https://quarkus.io/guides/validation#validation-and-localization

2.) Or forcing tests to use a default locale, by configuring it via `maven-surefire-plugin`: https://stackoverflow.com/questions/8901880/set-surefire-plugin-locale-in-maven-pom-file

--- a/integration-tests/hibernate-orm-rest-data-panache/src/main/resources/application.properties
+++ b/integration-tests/hibernate-orm-rest-data-panache/src/main/resources/application.properties
@@ -4,3 +4,6 @@ quarkus.datasource.jdbc.max-size=8
 quarkus.hibernate-orm.database.generation=drop-and-create
 quarkus.hibernate-orm.log.sql=true
 quarkus.hibernate-orm.sql-load-script=import.sql
+
+quarkus.default-locale=en_US
+quarkus.locales=en_US

